packages/grafana-ui/src/schema.ts

```diff
@@ -16,7 +16,7 @@ export {
   ScaleDistributionConfig,
   HideSeriesConfig,
   BarAlignment,
-  BarValueVisibility,
+  VisibilityMode as BarValueVisibility,
```

Comment:
is there a way to add `@deprecated` to this export? :thinking:

Reply:
:thinking: -- right now at least it won't be in docs

Reply:
Just to pre-empt a possible question, the answer to this question will be entirely on the TS side, it'd be quite circuitous to try to do this from CUE